src/main.rs

```diff
@@ -919,28 +922,52 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_function_call() {
+        parse_str(
+            r#"
+                fn test() {}
+                test()
+            "#,
+        );
+    }
+
+    #[test]
+    fn test_function_call_with_args() {
+        parse_str(
+            r#"
+                fn test(a int, b int) {}
+                test(1, 2)
+            "#
+        );
+    }
+
     #[test]
     fn test_variable() {
         parse_str(
             r#"
-                fn test() string {
-                    let foo = "1234"
-                }
+                let foo = "1234"
```

Comment:
I'm not sure if it's a good idea to allow creation of mutable global variables.  
A lot of programming languages have limitations to creating global mutable variables.  

I guess we can better change this to a `const` and limit it's assignment to only static data.
```suggestion
                const foo = "1234"
```